### DISCUSSION TITLE
CQI and SINR values in the "cars" scenario

### DISCUSSION THREAD

**Discussion by rimsayegh** [Unipisa/Simu5G#201] [Q&A]
Hello everyone,

I am currently engaged in a project involving the car scenario within Simu5G. I observed that altering the CQI value, either by increasing or decreasing it, did not correspondingly affect the "measuredSinrDl:mean" and "measuredSinrUl:mean" values. Typically, one would expect the SINR value to increase in tandem with the CQI value.

My question is: How are "measuredSinrDl:mean" and "measuredSinrUl:mean" implemented in Simu5G? And What is the problem in my results?

Thank you

**Comment by alemilan** [comment]
Hello,

Don't known if this can help you, but I inserted a log (writing in a file) directly in simu5g/src/stack/phy/ChannelModel/LteRealisticChannelModel.cc, within getSINR function. As I understood, every PKT pass through this function, HANDOVERPKT and DATAPKT, and the SINR is calculated for both. About CQI, I inserted a log in simu5g/src/stack/phy/layer/NRPhyUe.cc, within handleAirFrame function. 

There may be some adjusts to be done and this may be the worst approach, but worked for me. Hope this may be a path for you. 

Best Regards
